### PR TITLE
doc/edge-ai: update documentation @open sesame 07/15 11:22

### PR DESCRIPTION
**Pull request by myungjoo**
Add more explanation on edge-AI / among-device AI

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
@gichan-jang : the diagram is misleading. please redraw (cloud is not required by edge-AI although it can be "supplement".)

**Comment by gichan-jang** [comment]
> @gichan-jang : the diagram is misleading. please redraw (cloud is not required by edge-AI although it can be "supplement".)

I will update the diagram after this PR.

File: Documentation/edge-ai.md [hunk 1]
```diff
@@ -1,28 +1,42 @@
...
+Running AI at the ‘edge’ of the local network removes the requirement for the device to be connected to the internet or centralized servers like the cloud.
 Edge AI offers significant improvements as far as response speeds and data security.
-
 Executing AI close to the data source allows for processes like data creation and decision-making to take place in milliseconds, making Edge AI ideal for applications where near-instantaneous responses are essential.
 
-  * Reference : [On the Edge - How Edge AI is reshaping the future](https://www.samsung.com/semiconductor/newsroom/tech-trends/on-the-edge-how-edge-ai-is-reshaping-the-future/)
+
+Among-Device AI / Edge-AI can be often extended to conventional cloud AI services so that parts of data can be sent to clouds for further processing or data gathering for future neural network models.
+Techincally, for pipeline frameworks such as NNStreamer and GStreamer, they are not very different from among-device AI between embedded devices (exactly same code can be applied!).
```
Comment (helloahn) [inline Documentation/edge-ai.md:21]:
```suggestion
Technically, for pipeline frameworks such as NNStreamer and GStreamer, they are not very different from among-device AI between embedded devices (exactly same code can be applied!).
```

typo

File: Documentation/edge-ai.md [hunk 2]
```diff
@@ -1,28 +1,42 @@
...
+With on-device AI, different embedded devices including mobile phones, TVs, refrigerators, vacuum cleaners, and all the other sorts of deviecs have started running deep neural network models.
+Unlike conventional AI services running in cloud servers, on-device AI services are often limited by the limited computing resources of the given devices; they usually have limited size of memory, limited processing power, or limited energy, which varies per device.
+Moreover, such devices have different availability of data; each device may have different sensors or different sensing targets (e.g., view angles of cameras), whcih might be beneficial if a device has access to others.
 
-Running AI at the ‘edge’ of the local network removes the requirement for the device to be connected to the internet or centralized servers like the cloud.
 
+Among-Device AI (a.k.a. Edge-AI in some industry context) tries to mitigate such issues by connecting such devices and let them share data and processing power between them.
+In other words, Amond-Device AI / Edge-AI allows distributed computing for AI algorithms, which can distribute computing workloads and share data from different devices.
```
Comment (helloahn) [inline Documentation/edge-ai.md:14]:
```suggestion
In other words, Among-Device AI / Edge-AI allows distributed computing for AI algorithms, which can distribute computing workloads and share data from different devices.
```

typo

File: Documentation/edge-ai.md [hunk 3]
```diff
@@ -1,28 +1,42 @@
...
+# What is Edge-AI and Among-Device AI?
 
-However, running AI in the cloud has its disadvantages, including the fact that it requires an internet connection, and that performance can be impacted by bandwidth and latency limitations.
 
-Edge AI, also known as ‘on-device AI’, is a distributed computing paradigm that allows for AI algorithms to be run locally on a device, using the data produced by that device.
+With on-device AI, different embedded devices including mobile phones, TVs, refrigerators, vacuum cleaners, and all the other sorts of deviecs have started running deep neural network models.
+Unlike conventional AI services running in cloud servers, on-device AI services are often limited by the limited computing resources of the given devices; they usually have limited size of memory, limited processing power, or limited energy, which varies per device.
+Moreover, such devices have different availability of data; each device may have different sensors or different sensing targets (e.g., view angles of cameras), whcih might be beneficial if a device has access to others.
```
Comment (helloahn) [inline Documentation/edge-ai.md:10]:
```suggestion
Moreover, such devices have different availability of data; each device may have different sensors or different sensing targets (e.g., view angles of cameras), which might be beneficial if a device has access to others.
```

typo

**Comment by myungjoo** [comment]
Typo fixed.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**